tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1233,6 +1251,10 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
       }
     }
 
+    if (bitcast) {
```

Comment:
I think this should be `slice`, not `bitcast`.

Reply:
When the operations of both `bitcast` and `slice` are present, it indicates the presence of high rank input.